src/Client/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace InterviewApp.Client.Extensions;
+
+public static class StringExtensions
```

Comment:
You're definitely right on this one.
It might seem more readable or prettier but considering the fact you stated in different comment that other developers contributing to this repository might not know about these extension methods, it will be better to remove them. ☺

> Unless I'm blind and it has like 300 references. 😅 Always an option

And you're not blind, there are really no references for `IsNullOrEmpty`. I planned on using it later in case of need and I of course get the point you're stating. #YAGNI 😄